### PR TITLE
feat: revise frontend design

### PR DESCRIPTION
**Pull request by QiyuanMa**

### CODE REVIEW

File: uniswap-interface-stealth/src/theme/BackgroundComponent.tsx [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+    };
+  }, [scriptsLoaded.three, scriptsLoaded.vanta]);
+
+  return <div ref={vantaRef} style={{ height: '100vh' }}>{children}</div>;
+};
+
+export default BackgroundComponent;
+
+
+// import React, { useEffect, useRef, useState, ReactNode } from 'react';
```
Comment (eleminsoda) [inline uniswap-interface-stealth/src/theme/BackgroundComponent.tsx:75]: Can we remove the commented codes?

**Review by eleminsoda — COMMENTED** [review]
Do we want to note down what this PR has done and how the final output looks like in the description?

File: uniswap-interface-stealth/src/theme/BackgroundComponent.tsx (hunk 1 above)
Comment (n06ody) [reply uniswap-interface-stealth/src/theme/BackgroundComponent.tsx:75]: +1

File: uniswap-interface-stealth/src/theme/index.tsx [hunk 2]
```diff
@@ -21,6 +21,21 @@ const MEDIA_WIDTHS = {
   upToLarge: 1280
 }
 
+// 转换十六进制颜色为RGBA格式的函数
```
Comment (n06ody) [inline uniswap-interface-stealth/src/theme/index.tsx:24]: Think we can remove these auto-generated comments or replace with something meaningful

**Review by n06ody — CHANGES_REQUESTED** [review]

**Pull request merged**